wa/framework/execution.py

```diff
@@ -76,6 +76,11 @@ def spec_will_change(self):
         return self.current_job.spec.id != self.next_job.spec.id
 
     @property
+    def workload(self):
```

Comment:
The "current" is implied, as with `output`.  The reason the Job is explicitly `current_job` is because there is also a `previous_job` and a `next_job`. But (at the moment) there is no reason to access current/previous workload/output. If that changes, this will be renamed.